app/models/nearby_facility.rb

```diff
@@ -10,9 +10,15 @@ class << self
     def query(street_address: '', city: '', state: '', zip: '', **params)
       return NearbyFacility.none unless [street_address, city, state, zip].all?(&:present?)
 
-      waypoint = "#{street_address} #{city} #{state} #{zip}"
-      isochrone_response = request_isochrone(waypoint, params)
-      get_facilities_in_isochrone(params, isochrone_response)
```

Comment:
Are these methods still necessary?

Reply:
not anymore for the method that takes in an address as we now want to call an additional bing api to get the lat/lng and call the existing get nearby facilities by lat/lng method

Reply:
Awesome. Love seeing code removed